templates/rails_helper.rb

```diff
@@ -14,8 +14,9 @@ module Features
 
 RSpec.configure do |config|
   config.before(:each, js: true) do
-    page.driver.block_unknown_urls
+    page.block_unknown_urls = true
```

Comment:
You're right. Fixing the reference, but I am fairly sure I got this to pass when I ran `rake`, which is concerning.